### PR TITLE
More required information added to accessory directory

### PR DESCRIPTION
**Pull request by ellisrichardj**
This PR primarily adds required input files to the accessory directory.  The `auspice_config.json` provides information to correctly set up the display of information in `Auspice`.  `custom-colours.tsv` links specific colours to metadata from consistent display.  Currently, only host animals have been included, but this file can be expanded to any of the metadata used for filtering.  Both of these files are provided to the `augur export` command.

There are also a few minor changes to enable the correct merging of the locations files, and the addition of further outlier samples

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -28,6 +28,7 @@ params.pathTocsv = "$PWD/**/*FinalOut*.csv"
 params.metadata = "$PWD/movement.csv"
 params.locations = "$PWD/locations.csv"
 params.auspiceconfig = "$PWD/auspice_config.json"
+params.colours = "$projectDir/accessory/custom-colous.tsv"
```
[the remark this answers is not in the document]
Comment (ellisrichardj) [reply nextflow.config]: I think you're looking at an old commit - this has already been corrected

File: accessory/counties.tsv [hunk 2]
```diff
@@ -1,4 +1,4 @@
-LocationType	County	Lat	Long
+LocationType	CPH	Lat	Long
```
Comment (nick-pestell) [inline accessory/counties.tsv:1]: I'm sure you're probably right, I just wanted to question why this now `CPH`; looks like the entries are actually all still county.

**Review by nick-pestell — COMMENTED** [review]
Yeah, looks great. I had just one question regarding the `counties.tsv`

File: accessory/counties.tsv (hunk 2 above)
Comment (ellisrichardj) [reply accessory/counties.tsv:1]: You're right, they are all county - it was just about getting the CPH and Counties locations to merge properly

**Review by Hira-Naveed — APPROVED** [review]
Looking good Richard! Good to see the files being constructed in this way.

File: accessory/counties.tsv (hunk 2 above)
Comment (nick-pestell) [reply accessory/counties.tsv:1]: Sounds good.

**Pull request merged**